src/Drivers/S3.ts

```diff
@@ -259,7 +260,10 @@ export class S3Driver implements S3DriverContract {
           Key: location,
           Bucket: this.config.bucket,
           ...this.transformContentHeaders(options),
-        })
+        }),
+        {
+          expiresIn: string.toMs(options?.expiresIn || '15min') / 1000,
```

Comment:
Instead of manually dividing by one thousand, wouldn't it be better to add a new helper in [poppinss/utils](https://github.com/poppinss/utils)?

Reply:
Yup.

Reply:
Maybe wait and see if we need it at other places as well and then extract to a reusable helper?

Reply:
I don't really have an opinion on this. Tell me what you guys prefer and I'll do it 🙂

Reply:
Let's merge it for now!